### PR TITLE
Fixes #27, Fixes #44 : Add icon image, updated GUI and notified requirement of cert manager

### LINKED ISSUE #27: Documentation
Tell the user to install a certificate manager.

### LINKED ISSUE #44: Operator GUI look and feel
Add support for the Operator in GUI
Need a logo and the font size needs update, etc.

### PR DESCRIPTION
**Pull request by makon57**
This fixes #27, fixes #44 : Add icon, updated GUI and notified requirement of cert manager

### CODE REVIEW

File: l5-operator/config/manager/kustomization.yaml [hunk 1]
```diff
@@ -12,4 +12,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/rocrisp/controller.1.5
+  newName: quay.io/mkong/controller
```
Comment (skattoju) [inline l5-operator/config/manager/kustomization.yaml:15]: should probably push images to opdev

File: l5-operator/Makefile [hunk 2]
```diff
@@ -3,7 +3,7 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 0.0.1
+VERSION ?= 0.0.3
```
Comment (skattoju) [inline l5-operator/Makefile:6]: apparently this should always be 0.0.0 for the dev branch per discussion in slack.. ok for now but we can revisit

**Review by skattoju — APPROVED** [review]

**Comment by skattoju** [comment]
@makon57 you should squash you commits to a single commit

File: l5-operator/Makefile (hunk 2 above)
Comment (rocrisp) [reply l5-operator/Makefile:6]: I imagine we will reset this number to 0.0.1 ( default ) for now, it's fine.

File: l5-operator/config/manager/kustomization.yaml (hunk 1 above)
Comment (rocrisp) [reply l5-operator/config/manager/kustomization.yaml:15]: Agreed.

**Review by rocrisp — APPROVED** [review]

**Pull request merged**